test/unit/rules/no-mut-helper-test.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+
+const { message, generateMessageWithAlternative } = require('../../../lib/rules/no-mut-helper');
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const setterAlternative = 'ember-simple-set-helper$set';
```

Comment:
`$set` ?

why do we need to mention `ember-simple-set-helper` in error?

Reply:
good point, Alex. as Chris mentioned in his comment below, most Ember apps use standard syntax, not [Batman/Wall Street sytnax](https://github.com/rwjblue/ember-holy-futuristic-template-namespacing-batman) (where the helper name is prefixed with the addon name of the dependency).

Fixed - I have update the Pull Request to only use `{{set}}` and omit the addon name prefix of the dependency.

Thanks for the suggestion.